.github/workflows/deploy-gh-pages.yml

```
@@ -0,0 +1,31 @@
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build Stencil app
+        run: npm run build
+
+      - name: Deploy to GitHub Pages
+        uses: peaceiris/actions-gh-pages@v4
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: /app/www
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)